windows.c

```diff
@@ -97,7 +101,7 @@ int initialize(INTERNET_PER_CONN_OPTION_LIST* options) {
 int query(INTERNET_PER_CONN_OPTION_LIST* options) {
   DWORD dwBufferSize = sizeof(INTERNET_PER_CONN_OPTION_LIST);
   if(!InternetQueryOption(NULL, INTERNET_OPTION_PER_CONNECTION_OPTION, options, &dwBufferSize)) {
-    reportWindowsError("Querying options");
+    reportWindowsError("querying options", options->pszConnection);
```

Comment:
How about `reportWindowsError("querying options", options->pszConnection || "LAN");`? There are two cases the second parameter is intentionally NULL.